sentry_sdk/integrations/starlette.py

```diff
@@ -362,13 +363,19 @@ def patch_middlewares():
 
     if not_yet_patched:
 
-        def _sentry_middleware_init(self, cls, **options):
-            # type: (Any, Any, Any) -> None
+        def _sentry_middleware_init(self, cls, *args, **kwargs):
+            # type: (Any, Any, Any, Any) -> None
             if cls == SentryAsgiMiddleware:
-                return old_middleware_init(self, cls, **options)
+                if StarletteIntegration.starlette_version < (0, 35):
+                    return old_middleware_init(self, cls, **kwargs)
+                else:
+                    return old_middleware_init(self, cls, *args, **kwargs)
 
             span_enabled_cls = _enable_span_for_middleware(cls)
-            old_middleware_init(self, span_enabled_cls, **options)
+            if StarletteIntegration.starlette_version < (0, 35):
+                old_middleware_init(self, span_enabled_cls, **kwargs)
+            else:
+                old_middleware_init(self, span_enabled_cls, *args, **kwargs)
```

Comment:
Same comment applies here, as well.
```suggestion
            old_middleware_init(self, span_enabled_cls, *args, **kwargs)
```

Reply:
done. see other comment.